08-single-page-web-apps/tdd/readme.md

```diff
@@ -192,75 +211,64 @@ A "suite" is the highest-level container in our test file.
 
 **2. The Specs**
 
+A suite contains **specs**. These are the `it` statements.
+
+In the "spec," we target a specific part of the suite.
+
+To make the specs actually do something, you need to pass a function as a second argument to each one. This function is what Jasmine will run to check whether the spec passes.
+
+For the snowman builder, the result would be this:
+
 ```js
 describe( "A snowman", function(){
 
-  //My winter wonderland is a friendly place, so I want each snowman to have a name.
```

Comment:
I miss these comments.  I think they helped clarify, motivate, and connect to the earlier planning portion.  A chain of evidence.